specification/workloads/resource-manager/readme.go.md

```diff
@@ -23,6 +23,7 @@ output-folder: $(go-sdk-folder)/$(module-name)
 batch:
   - tag: package-2021-12-01-preview
   - tag: package-preview-2022-11
+  - tag: package-2023-04
```

Comment:
Please follow https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/samplefiles/readme.go.md to configure the go SDK.  Thanks

Reply:
Fixed